LabelGenerator.py

```diff
@@ -357,14 +369,14 @@ def draw_resistor_colorcode(
                                  y + border,
                                  stripe_width,
                                  height - 2 * border,
-                                 stripe_value)
+                                 stripe_value, tolerance_color)
 
         draw_resistor_stripe(c,
```

Comment:
This doesn't have to be a new argument, just define a mapping between the tolerance and the resistor values. Like, pass `1` for brown instead of `-3`. That way we avoid re-defining colors, and things like gold/silver looks consistent with the other stripes.